presidio_evaluator/evaluation/evaluator.py

```diff
@@ -508,35 +584,97 @@ def generate_graph(title, tokens_df):
                         showgrid=False,  # Removes X-axis grid lines
                     ),
                     yaxis=dict(
-                        title=f"Tokens",
+                        title="Tokens",
                         linecolor="#BCCCDC",  # Sets color of X-axis line
                         showgrid=False,  # Removes X-axis grid lines
                     ),
+                    height=10 * len(tokens_df),
                 )
                 fig.update_layout(yaxis={"categoryorder": "total ascending"})
-                fig.show()
+                fig.show(save_as)
```

Comment:
Maybe adding error handling here to handle unexpected value of save_as paramter?

Reply:
Also I'm thinking in the automated pipeline, the end-user might prefer to save the plot as a file rather than visualizing it directly. Therefore, it could be beneficial to include an option like fig.write_image('filename.png')

Reply:
One way to do this is just to return the `fig` object, which allows the user to take it and save it. Another is to add a parameter with a `save_as` path. Which one would be better in your perspective?

Reply:
I prefer to return the fig object.

Reply:
I've updated the Plotter class, and each plot function does three things: display the fig, save it (if the user wants to save), and return the fig. In one case, it returns a tuple of figs as the function creates three different figures (for f2, recall, precision)